### PR TITLE
Adding Jenkins mvn examples

### PR DESCRIPTION
**Pull request by ghost**
@samq-ws @tidharm 
Appreciate your review.

### CODE REVIEW

File: CI-CD/Jenkins/Maven/Jenkins_pipeline.groovy [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        sh 'mvn clean install -DskipTests'
+      }
+    }
+    stage('Download WS Script') {
+      steps {
+                script {
+                    if (fileExists('./wss-unified-agent.jar')) {
+                        echo "File already exists"
+                    } else {
+                            sh 'curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar'
```
Comment (samq-ws) [inline CI-CD/Jenkins/Maven/Jenkins_pipeline.groovy:27]: use amazon download link

File: CI-CD/Jenkins/Jenkins_freestyle.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+echo "Downloading WS"
+if ! [ -f ./wss-unified-agent.jar ]; then
+  curl -fSL -R -JO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+fi
+echo "Exceute WS"
+export WS_APIKEY=${APIKEY} #Taken from Jenkins Global Environment Variables
+export WS_WSS_URL="https://saas.whitesourcesoftware.com/agent"
```
Comment (samq-ws) [inline CI-CD/Jenkins/Jenkins_freestyle.sh:7]: add userkey as it is a best practice

File: CI-CD/Jenkins/Jenkins_freestyle.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+echo "Downloading WS"
+if ! [ -f ./wss-unified-agent.jar ]; then
+  curl -fSL -R -JO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
```
Comment (samq-ws) [inline CI-CD/Jenkins/Jenkins_freestyle.sh:3]: use amazon s3 download link

**Comment by tabac-ws** [comment]
merged with CodeFresh

**Pull request closed without merging**